src/script.js

```diff
@@ -156,6 +154,10 @@ class SuppDapp extends HTMLElement {
             newTic($('.supp-title').value)
             $('.supp-title').value = '';
           })
+          $('.verify-site-button').addEventListener('click', async () => {
+            const siteVerified = await connectSnap(); 
```

Comment:
Save this variable to state and if true, disable the button and change its text to verified maybe?

Reply:
Something like that. Maybe keep the button enabled in case they want to re-check the data?